main.go

```diff
@@ -9,24 +9,27 @@ import (
 	"strings"
 	"time"
 
+	"github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/session"
+
 	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/command/rubycommand"
 	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
 	"github.com/bitrise-io/go-utils/pathutil"
 	"github.com/bitrise-io/go-utils/retry"
 	"github.com/bitrise-tools/go-steputils/input"
+	"github.com/bitrise-tools/go-steputils/tools"
 	"github.com/kballard/go-shellquote"
 )
 
 // ConfigsModel ...
 type ConfigsModel struct {
```

Comment:
update the model to use the new stepconf package and use that instead

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/615bd5f8c8369ebd51f43372703021968554ac84